{"name": "数据库中查不到用户", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'true' == '-1'\n  - -1\n  + true", "trace": "pyfixture = ({'custid': 104401903883, 'email': '120666@qq.com', 'mobile_phone': 18888118881, 'result_format': 'json'}, ('-1', '数据库中查不到用户'))\n\n    @allure.title('{pyfixture[1][1]}')\n    @pytest.mark.check_email\n    @pytest.mark.flaky(reruns=1,reruns_delay=10)\n    def test_CheckEmail(pyfixture):\n    \n        #请求\n        url=Check_Email.url\n        data=pyfixture[0]\n        res=request(url=url,data=data)\n        #print(data)\n    \n        #返回码检查\n        if data['email'] !='':\n>            assert res['if_exist']==pyfixture[1][0]\nE            AssertionError: assert 'true' == '-1'\nE              - -1\nE              + true\n\ntestcase\\Loginapi\\Check_Email_test.py:56: AssertionError"}, "parameters": [{"name": "pyfixture", "value": "({'email': '120666@qq.com', 'result_format': 'json', 'mobile_phone': 18888118881, 'custid': 104401903883}, ('-1', '数据库中查不到用户'))"}], "start": 1619081388647, "stop": 1619081388680, "uuid": "01898be4-a177-43d0-9882-c600e6e8dec0", "historyId": "852b8fd0089b6a877dbfde8f4a77c310", "testCaseId": "90221c019d6c5a5af9cb6011e67ab3fe", "fullName": "testcase.Loginapi.Check_Email_test#test_CheckEmail", "labels": [{"name": "tag", "value": "flaky(reruns=1, reruns_delay=10)"}, {"name": "tag", "value": "check_email"}, {"name": "parentSuite", "value": "testcase.Loginapi"}, {"name": "suite", "value": "Check_Email_test"}, {"name": "host", "value": "A06P-A90F12DAA"}, {"name": "thread", "value": "1088-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.Loginapi.Check_Email_test"}]}